iterators_task_1/student/include/impl.h

```diff
@@ -10,4 +12,21 @@
  *
  * @warning YOU MUST USE STRING STREAMS AND STREAM ITERATORS!
  */
-std::string reverseWords(const std::string& input);
+std::string reverseWords(const std::string& input){
+    std::stringstream in{input};
+    std::stringstream out{};
```

Comment:
please always declare variables near the point of usage, don't declare everything in one place (see cpp core guidelines)